src/gevent/tests/test__issue1864.py

```diff
@@ -0,0 +1,21 @@
+import math
+import unittest
+
+from gevent.testing import PY39
+
+
+class TestSubnormalFloatsAreNotDisabled(unittest.TestCase):
+    """
+    Enabling the -Ofast compiler flag resulted in subnormal floats getting
+    disabled the moment when gevent was imported. This impacted libraries
+    that expect subnormal floats to be enabled.
+    """
+    @unittest.skipUnless(PY39, "Need math.nextafter()")
+    def test_subnormal_is_not_zero(self):
+        import gevent
+
+        assert math.nextafter(0, 1) != 0
```

Comment:
There's a less elegant way to test this on Python 3.8 and earlier too - [`sys.float_info.min` is the minimum *normalized* float](https://docs.python.org/3/library/sys.html#sys.float_info), so you can `assert (sys.float_info.min / 2) > 0` and with a block comment it's not even too confusing.

Reply:
@Zac-HD, that's great. Makes perfect sense, and somewhat clearer than my suggestion.

Reply:
Thanks for the suggestion. Updated the test to use `sys.float_info.min` (no `skipUnless` anymore!), and confirmed that [it fails as expected when gevent is built with `-Ofast`](https://github.com/ento/gevent/actions/runs/3123769165).